### PR TITLE
Request to Add Source Mainnet

### PR DESCRIPTION
**Pull request by Source-Protocol-Cosmos**
Updated from previous PR.

www.sourceprotocol.io

https://www.coingecko.com/en/coins/source

https://explorer.tcnetwork.io/source

### CODE REVIEW

**Comment by HeesungB** [comment]
@Source-Protocol-Cosmos Hello.
Could you remove `coingekcoId` field? and You can add after token price shown.

**Comment by Source-Protocol-Cosmos** [comment]
> @Source-Protocol-Cosmos Hello. Could you remove `coingekcoId` field? and You can add after token price shown.

Done.

File: cosmos/source.json [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+      "coinDenom": "SOURCE",
+      "coinMinimalDenom": "usource",
+      "coinDecimals": 6
+    }
+  ],
+  "feeCurrencies": [
+    {
+      "coinDenom": "SOURCE",
+      "coinMinimalDenom": "usource",
+      "coinDecimals": 6
```
[the remark this answers is not in the document]
Comment (Source-Protocol-Cosmos) [reply cosmos/source.json]: done

[the remark this answers is not in the document]
Comment (Source-Protocol-Cosmos) [reply cosmos/source.json]: What are we missing here? How can we get this moving forward?

Thank you for your time and help

[the remark this answers is not in the document]
Comment (HeesungB) [reply cosmos/source.json]: There is no missing! I will merge it!

**Pull request merged**